### PR TITLE
LoadImages reformat

### PR DESCRIPTION
**Pull request by vchernys**

### CODE REVIEW

File: cellprofiler/modules/loadimages.py [hunk 1]
```diff
@@ -436,199 +463,241 @@ def example_path_fn():
...
+| (?P<Date>                                           | Name the captured field *Date*                                                                                                                                                 |
++-----------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| .*                                                  | Capture as many characters that follow                                                                                                                                         |
++-----------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| [\\\\\\\\/]                                         | Discard the slash/backslash character                                                                                                                                          |
```
Comment (bethac07) [inline cellprofiler/modules/loadimages.py]: Please add 4 more spaces in the first column of this line to get the table to format correctly.

File: cellprofiler/modules/loadimages.py [hunk 2]
```diff
@@ -436,199 +463,241 @@ def example_path_fn():
...
+
+For instance, a researcher uses folder names with the date and
+subfolders containing the images with the run ID (e.g.,
+*.2009_10_02/1234*) The following regular expression will capture
+the plate, well, and site in the fields *Date* and *Run*:
+
++-----------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| .*[\\\\\\/](?P<Date>.*)[\\\\\\\\/](?P<Run>.*)$      |
++-----------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| .*[\\\\\\\\/]                                       | Skip characters at the beginning of the pathname until either a slash (/) or backslash (\\\\) is encountered (depending on the operating system)                               |
```
Comment (bethac07) [inline cellprofiler/modules/loadimages.py]: Please add 4 more spaces in the first column of this line to get the table to format correctly.

File: cellprofiler/modules/loadimages.py [hunk 3]
```diff
@@ -436,199 +463,241 @@ def example_path_fn():
...
+your expression. The syntax *(?<fieldname>expr)* will extract whatever
+matches *expr* and assign it to the image’s *fieldname* measurement.
+
+For instance, a researcher uses folder names with the date and
+subfolders containing the images with the run ID (e.g.,
+*.2009_10_02/1234*) The following regular expression will capture
+the plate, well, and site in the fields *Date* and *Run*:
+
++-----------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| .*[\\\\\\/](?P<Date>.*)[\\\\\\\\/](?P<Run>.*)$      |
```
Comment (bethac07) [inline cellprofiler/modules/loadimages.py]: Please align the end line here with the end line of the table

File: cellprofiler/modules/loadimages.py (hunk 2 above)
Comment (bethac07) [reply cellprofiler/modules/loadimages.py]: Also add two spaces to the second column of this line.

File: cellprofiler/modules/loadimages.py [hunk 4]
```diff
@@ -436,199 +463,241 @@ def example_path_fn():
...
+the plate, well, and site in the fields “Plate”, “Well”, and “Site”:
+
++----------------------------------------------------------------+------------------------------------------------------------------+
+| ^(?P<Plate>.*)_(?P<Well>[A-P][0-9]{1,2})_s(?P<Site>[0-9])   |
++----------------------------------------------------------------+------------------------------------------------------------------+
+| ^                                                              | Start only at beginning of the file name                         |
++----------------------------------------------------------------+------------------------------------------------------------------+
+| (?P<Plate>                                                     | Name the captured field *Plate*                                  |
++----------------------------------------------------------------+------------------------------------------------------------------+
+| .*                                                            | Capture as many characters as follow                             |
```
Comment (bethac07) [inline cellprofiler/modules/loadimages.py]: Please add 1 more space in the first column of this line  to get the table to format correctly.

File: cellprofiler/modules/loadimages.py [hunk 5]
```diff
@@ -436,199 +463,241 @@ def example_path_fn():
...
++----------------------------------------------------------------+------------------------------------------------------------------+
+| _                                                             | Discard the underbar separating plate from well                  |
```
Comment (bethac07) [inline cellprofiler/modules/loadimages.py]: Please add 1 more space in the first column of this line to get the table to format correctly.

File: cellprofiler/modules/loadimages.py [hunk 6]
```diff
@@ -436,199 +463,241 @@ def example_path_fn():
...
++----------------------------------------------------------------+------------------------------------------------------------------+
+| (?P<Well>                                                      | Name the captured field *Well*                                   |
++----------------------------------------------------------------+------------------------------------------------------------------+
+| [A-P]                                                          | Capture exactly one letter between A and P                       |
++----------------------------------------------------------------+------------------------------------------------------------------+
+| [0-9]{1,2}                                                     | Capture one or two digits that follow                            |
++----------------------------------------------------------------+------------------------------------------------------------------+
+| _s                                                            | Discard the underbar followed by *s* separating well from site   |
```
Comment (bethac07) [inline cellprofiler/modules/loadimages.py]: Please add 1 more space in the first column of this line  to get the table to format correctly.

File: cellprofiler/modules/loadimages.py [hunk 7]
```diff
@@ -436,199 +463,241 @@ def example_path_fn():
...
+
+For instance, a researcher uses plate names composed of a string of
+letters and numbers, followed by an underscore, then the well,
+followed by another underscore, followed by an “s” and a digit
+representing the site taken within the well (e.g.,
+*TE12345_A05_s1.tif*). The following regular expression will capture
+the plate, well, and site in the fields “Plate”, “Well”, and “Site”:
+
++----------------------------------------------------------------+------------------------------------------------------------------+
+| ^(?P<Plate>.*)_(?P<Well>[A-P][0-9]{1,2})_s(?P<Site>[0-9])   |
```
Comment (bethac07) [inline cellprofiler/modules/loadimages.py]: Please align the end line here with the end line of the table

File: cellprofiler/modules/loadimages.py (hunk 2 above)
Comment (vchernys) [reply cellprofiler/modules/loadimages.py]: I think I figured it out but I am having an issue updating LoadImages...

File: cellprofiler/modules/insurance [hunk 8]
```diff
@@ -0,0 +1,95 @@
++----------------------------------------------------------------------------------------------------------------------+
```
Comment (bethac07) [inline cellprofiler/modules/insurance]: Please remove this extra file.

**Pull request merged**